cmd/addons.go

```diff
@@ -21,23 +21,22 @@ var (
 `,
 		Action: func(c *cli.Context) error {
 			currentApp := detect.CurrentApp(c)
-			var err error
-			if c.Args().Len() == 0 {
-				err = addons.List(c.Context, currentApp)
-			} else {
-				cli.ShowCommandHelp(c, "addons")
+			if c.Args().Len() > 0 {
+				return cli.ShowCommandHelp(c, "addons")
```

Comment:
With the change explained here (https://github.com/Scalingo/cli/pull/835/files#r1056545792), this leads to the following output:

```
 ✘ emichon@biniou  ~/Documents/Scalingo/golang/src/github.com/Scalingo/cli   fix/834/addons-add_help  go build -o scalingo-cli ./scalingo && ./scalingo-cli --app biniou addons toto
No help topic for 'addons'
```